components/Banner.js

```diff
@@ -0,0 +1,389 @@
+                    </Button>
+
+                    <StyledMenu
+                        id="nodes-menu"
+                        anchorEl={anchorEl}
+                        keepMounted
+                        open={Boolean(anchorEl)}
+                        onClose={this.handleNodeClose}
+                        anchorOrigin={{
+                            horizontal: 'left', 
+                            vertical: 'bottom'
+                        }}
+                        transformOrigin={{
+                            horizontal: '40px'
+                        }}
+                        PaperProps={{
+                            style: {
+                                top: '36px'
+                            }
+                        }}
+                    >
+                        <StyledMenuItem onClick={() => {this.handleNodeSelect('https://pds-atmospheres.nmsu.edu')}}>
+                            <Link
+                                className={classes.menuLink}
+                                href='https://pds-atmospheres.nmsu.edu'
+                                rel='noopener'
+                            >
+                                <ListItemText className={classes.listItemText} primary='Atmospheres (ATM)'/>
+                            </Link>
+                        </StyledMenuItem>
+                        <StyledMenuItem onClick={() => {this.handleNodeSelect('https://pds-geosciences.wustl.edu')}}>
+                            <Link
+                                className={classes.menuLink}
+                                href='https://pds-geosciences.wustl.edu'
+                                rel='noopener'
+                            >
+                                <ListItemText className={classes.listItemText} primary='Geosciences (GEO)'/>
+                            </Link>
+                        </StyledMenuItem>
+                        <StyledMenuItem onClick={() => {this.handleNodeSelect('https://pds-imaging.jpl.nasa.gov')}}>
+                            <Link
+                                className={classes.menuLink}
+                                href='https://pds-imaging.jpl.nasa.gov'
+                                rel='noopener'
+                            >
+                                <ListItemText className={classes.listItemText} primary='Cartography and Imaging Sciences (IMG)'/>
+                            </Link>
+                        </StyledMenuItem>
+                        <StyledMenuItem onClick={() => {this.handleNodeSelect('https://naif.jpl.nasa.gov/naif')}}>
+                            <Link
+                                className={classes.menuLink}
+                                href='https://naif.jpl.nasa.gov/naif'
+                                rel='noopener'
+                                inputRef={this.textFieldRef}
```

Comment:
This seems like an unused boilerplate property and it's throwing this warning for me:
`
Warning: React does not recognize the `inputRef` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `inputref` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
`